### PR TITLE
modified the Quarkus operator SDK version and tutorial too

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
modified the Quarkus operator SDK vesion and tutorial too

### CODE REVIEW

File: pkg/quarkus/v1alpha/scaffolds/internal/templates/pomxml.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ const pomxmlTemplate = `<?xml version="1.0" encoding="UTF-8"?>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <fabric8-client.version>5.7.2</fabric8-client.version>
-    <quarkus-sdk.version>2.0.0.Beta5</quarkus-sdk.version>
+    <quarkus-sdk.version>1.9.5</quarkus-sdk.version>
```
[the remark this answers is not in the document]
Comment (laxmikantbpandhare) [reply pkg/quarkus/v1alpha/scaffolds/internal/templates/pomxml.go:61]: That's the latest version I found on `Quarkus Operator SDK`.

File: docs/tutorial.md [hunk 2]
```diff
@@ -200,24 +176,35 @@ There are a couple of ways to create the CRD. You can either create the file
...
+├── kubernetes.yml
+└── memcacheds.cache.example.com-v1.yml
+
+0 directories, 3 files
+```
+
+The content of the `memcacheds.cache.example.com-v1.yml` file is as shown below.
 
 ```
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
```
Comment (metacosm) [inline docs/tutorial.md:199]: I'm not sure how you generated this CRD but it doesn't look like it got generated properly. Perhaps it got generated before adding fields to `MemcachedSpec` and `MemcachedStatus` classes?

Comment (laxmikantbpandhare) [reply docs/tutorial.md:199]: Oh, yes! absolutely. I ran `mvn clean install` just after `create api` command. I will update the code and re-generate it thru compilation.

**Review by tlwu2013 — COMMENTED** [review]
I thought we don't need these manual steps in the README to enable java-operator-plugins?
Shall we remove those unnecessary steps?

File: docs/tutorial.md [hunk 3]
```diff
@@ -544,12 +505,6 @@ Now we have a `createOrUpdateResource` method. It calls
 `createMemcachedDeployment` which we have implemented above. In the next section
 we will discuss the deletion of the resource.
```
Comment (tlwu2013) [inline docs/tutorial.md:506]: This line regarding the "deletion" section needs to go as well.

Comment (tlwu2013) [reply docs/tutorial.md:506]: The sentence under " labelsForMemcached" section:
> Below the `deleteResource` method add the following
helper:

also needs to be removed

**Comment by tlwu2013** [comment]
Per chat @laxmikantbpandhare, I noticed there are a few deps are missing in the `MemcachedController.java` (and some in pom?) that causes build errors. 

Please help take a look and include them into the tutorials too, thanks!

File: docs/tutorial.md (hunk 3 above)
Comment (laxmikantbpandhare) [reply docs/tutorial.md:506]: Updated as `createOrUpdateResource`

**Comment by laxmikantbpandhare** [comment]
> Per chat @laxmikantbpandhare, I noticed there are a few deps are missing in the `MemcachedController.java` (and some in pom?) that causes build errors.
> 
> Please help take a look and include them into the tutorials too, thanks!

@tlwu2013 Updated the doc, please take a look it.

**Review by tlwu2013 — APPROVED** [review]
Verified build pass, Operator successfully deployed and running on a cluster, Operand instances spinning up correctly.

LGTM!  Thank you @laxmikantbpandhare!

**Comment by laxmikantbpandhare** [comment]
> Verified build pass, Operator successfully deployed and running on a cluster, Operand instances spinning up correctly.
> 
> LGTM! Thank you @laxmikantbpandhare!

@tlwu2013 thank you for helping with verification.

**Pull request merged**